src/components/Checkout/KivaCreditGuestPayment.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<div>
+		<div v-if="isGuestCheckout" id="guest-checkout">
+			<label class="input-label tw-font-medium tw-block tw-my-2" for="email">
+				What is your email?
+			</label>
+			<div class="input-label tw-text-small tw-text-secondary tw-block tw-my-2" for="email">
+				Kiva will share your information with {{ promoName ? promoName : 'your company' }}
+				to let them know you’ve redeemed your credits
+			</div>
+			<kv-text-input
+				type="email"
+				name="email"
+				v-model="email"
+				ref="email"
+				data-testid="basket-guest-email-input"
+				id="email"
+				class="data-hj-suppress tw-mb-2 tw-w-full"
+				@focus="$kvTrackEvent(
+					'basket',
+					'click-email-receipt-field',
+					'Where should we email your receipt?'
+				)"
+			/>
+			<p v-if="$v.email.$error" class="input-error tw-text-danger tw-text-base tw-mb-2">
+				Valid email required.
```

Comment:
One error message has a period, the other doesn't, is that expected?

Reply:
Alright I added a period to the other sentence.